theme/views.py

```diff
@@ -261,8 +261,11 @@ def signup_verify(request, uidb36=None, token=None):
 
 
 @login_required
-def update_user_profile(request):
-    user = request.user
+def update_user_profile(request, profile_user_id):
+    if request.user.is_superuser:
+        user = User.objects.get(id=profile_user_id)
```

Comment:
is the attribute is_superuser guaranteed to exist on the user object or is a safe .get() more appropriate?

Reply:
Yes, is_superuser is indeed guaranteed to exist on the user object, however, I think it'd be safer to make sure request.user is not None. I'll add a check for this.

Reply:
I took my last comment back. This view function has a ```login_required``` decorator, so request.user is guaranteed not None, so no check for None is needed here.